ssds/blobstore/local.py

```diff
@@ -52,6 +53,14 @@ def put(self, data: bytes):
         with open(self._path, "wb") as fh:
             fh.write(data)
 
+    @catch_blob_not_found
+    def copy_from(self, src_blob: "LocalBlob"):
+        """
+        Intra-cloud copy
+        """
+        assert isinstance(src_blob, type(self))
+        shutil.copyfile(src_blob._path, self._path)
```

Comment:
Should this simply return on the edge case where `src_blob._path == self._path`?

Reply:
Will add.